app/src/main/res/drawable/splash_background.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layer-list xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <item android:drawable="@color/blue_dark"/>
+
+    <item
+        android:height="200dp"
```

Comment:
Size is too big, make it default one, remove hight and width

Reply:
@therajanmaurya This is the screen on my phone on having default dimens.
![screenshot_2018-06-03-11-34-39-372_org mifos mobilebanking](https://user-images.githubusercontent.com/22195621/40884032-6bbf9ae8-6728-11e8-80b8-5b112e6e9c20.png)